### ISSUE TITLE
Use locale from `LocaleResolver` when `ui_locales` is empty in OIDC

### ISSUE THREAD

**Issue by morki** [micronaut-projects/micronaut-security#893]
### Feature description

I think this does make sense, because almost everytime user is logging in, he wants to use login form in the locale that he is already using in the application. What do you think @sdelamo? I can create PR for this.

**Comment by sdelamo** [comment]
Yes, PR is welcome. Do you have any authorization server in mind to test it (Google, Okta. e.g.) ?

**Comment by morki** [comment]
We use Keycloak, do you think we need to you use real instance of authorization server in test?

**Comment by sdelamo** [comment]
We have tests which use Keycloak via test container in case you want to check them.

**Comment by morki** [comment]
Thank you, will check them.
